bin/upgrading/en.dat

```diff
@@ -3983,7 +3983,7 @@ taglib.error.access=You are not allowed to tag entries in this journal.
 
 taglib.error.add=You are not allowed to create new tags for this journal; the entry was not tagged with [[tags]].
 
-taglib.error.delete=You are not allowed to delete tags from entries for this journal; the entry is still tagged with [[tags]].
+taglib.error.delete2=You are not allowed to delete tags from entries for this journal; the entry is still tagged with [[tags]].  Any tags you tried to add have not been added.
```

Comment:
I agree, but I think with that phrasing I'd expect the list of tags to be the entry's full list, where currently it's just the ones you tried to remove - maybe that's an improvement in any case. So:

> You are not allowed to delete tags from entries for this journal; any tags you tried to add have not been added.
>
> The entry is still tagged with [[all_of_its_tags]].